cli/crates/cli/tests/graphql/default/create.graphql

```diff
@@ -3,17 +3,20 @@ mutation {
     input: {
       title: "My todo list"
       todos: [
-        { create: { title: "My first todo!", complete: true } }
-        { create: { title: "My second todo!", complete: false } }
+        {create: {title: "My first todo!", complete: true}}
+        {create: {title: "My second todo!", complete: false}}
```

Comment:
Those extra spaces are added by prettier (there's a config in the root folder). Let's stick to the formatter.

Reply:
We need to add it to the CI but yes, we should be using prettier

Reply:
Ok, fixing it! thx